### PR TITLE
use sbt-version-policy and declare versionScheme

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by SethTisue** [comment]
The Scala 3 jobs are hitting https://github.com/scalacenter/sbt-version-policy/issues/62

and the Scala 2 jobs are getting

```
[info] scalacheck: mimaPreviousArtifacts is empty, not analyzing binary compatibility.
[error] sbt.librarymanagement.ResolveException: Error downloading default:scala-parallel-collections_2.13:1.0.1
[error]   Not found
[error]   not found: /home/travis/.ivy2/localdefault/scala-parallel-collections_2.13/1.0.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/default/scala-parallel-collections_2.13/1.0.1/scala-parallel-collections_2.13-1.0.1.pom
[error] 	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:258)
```

@julienrf note the incorrect URL — it's looking in `default/scala-parallel-collections_2.13` when it should be `org/scala-lang/modules/scala-parallel-collections_2.13`. I could use your help here. is this is a bug in sbt-version-policy?

File: build.sh [hunk 1]
```diff
@@ -47,4 +47,4 @@ export CI_SNAPSHOT_RELEASE="${projectPrefix}publish"
 # for now, until we're confident in the new release scripts, just close the staging repo.
 export CI_SONATYPE_RELEASE="; sonatypePrepare; sonatypeBundleUpload; sonatypeClose"
 
-sbt clean ${projectPrefix}test core/headerCheck ${projectPrefix}publishLocal $releaseTask
+sbt clean ${projectPrefix}test versionPolicyCheck core/headerCheck ${projectPrefix}publishLocal $releaseTask
```
Comment (julienrf) [inline build.sh:50]:
```suggestion
sbt clean ${projectPrefix}test core/versionPolicyCheck core/headerCheck ${projectPrefix}publishLocal $releaseTask
```

**Review by julienrf — COMMENTED** [review]
>  it's looking in `default/scala-parallel-collections_2.13` when it should be `org/scala-lang/modules/scala-parallel-collections_2.13`. I could use your help here. is this is a bug in sbt-version-policy?

Maybe, yes, or it could be an interaction between sbt-scala-module and sbt-version-policy, since both set MiMa settings.

It is not clear to me how the root project is defined, in this repository, but it’s `organization` setting is `default`, instead of `org.scala-lang.modules`.

A workaround is to invoke `versionPolicyCheck` only on the `core` project, which is the only one being published here.

File: build.sh (hunk 1 above)
Comment (SethTisue) [reply build.sh]: that works. but also, I've opened https://github.com/scalacenter/sbt-version-policy/issues/63

**Pull request merged**